src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala

```diff
@@ -353,12 +382,6 @@ trait OmmersPoolBuilder {
   lazy val ommersPool: ActorRef = system.actorOf(OmmersPool.props(blockchain, ommersPoolSize))
 }
 
-trait ValidatorsBuilder {
```

Comment:
This deletion causes a compilation error in snappy. Check `io.iohk.ethereum.snappy.Prerequisites.components`.